doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -37,20 +37,20 @@ depicts a high level view of the software profile features that apply to each
 instance profile (Basic and High Performance). For more information on the
 instance profiles please refer to :ref:`ref_model:chapters/chapter04:profiles`.
 
-.. image:: ../../../ref_model/figures/RM-ch05-sw-profile.png
-   :alt: "Figure 5-3 (from RM): NFVI softwareprofiles"
+.. figure:: ../../../ref_model/figures/RM-ch05-sw-profile.png
+   :alt: (from RM): NFVI softwareprofiles
+   :name: (from RM): NFVI software profiles
 
+   (from RM): NFVI software profiles
 
-**Figure 5-3 (from RM):** NFVI softwareprofiles
+In addition, the RM Figure (:numref:`(from RM): NFVI hardware profiles and host associated capabilities` shown below)
```

Comment:
This "numref" will give the reference of the figure in this text not Figure 5-4 from RM. 

@collivier @gkunz Suggestions? would

     :numref:`ref_model:figures/Cloud Infrastructure Hardware Profiles and host associated capabilities`

work?

Reply:
@pgoyal01, I cannot provide an answer unfortunately, but only a concern: I am wondering if Sphinx could determine the figure number when RA is built separately.